docs/github/builder.md

```diff
@@ -358,8 +358,8 @@ jobs:
         projectPath:
           - path/to/your/project
         unityVersion:
-          - 2019.2.11f1
-          - 2019.3.0f1
+          - 2019.4.11f1
+          - 2020.2.0f1
```

Comment:
These versions are actually not supported in v1. Latest gableroux images are no longer being built. I really think it's better to wait with updating all these docs for versioning to be in, as we won't have so many people verifying the correctness of the changes anymore.

Reply:
Sure, we can keep this PR open until after versioning is complete, and then apply this PR to v2 of the docs as appropriate?

Reply:
Sounds good!

Reply:
Versioning is in. Please leave v1 docs as they are as an historical artifact :)

Note that I also kept the old naming of the repository there, so that people have a way of finding out that it's the same thing by switching to the earlier version. Afterall, the move towards v2 is what made me move the repos to game-ci org recently, there most people that are upgrading will still be using the repo with my name in it.